app.json

```diff
@@ -5,6 +5,15 @@
     "postdeploy": "bundle exec rake db:setup"
   },
   "env": {
+    "DATABASE_URL": {
+      "required": true
+    },
+    "HEROKU_POSTGRESQL_CYAN_URL": {
+      "required": true
+    },
+    "HEROKU_POSTGRESQL_CYAN_URL": {
+      "required": true
+    },
```

Comment:
Probably doesn't make a difference, but this looks duplicated.

```
    "HEROKU_POSTGRESQL_CYAN_URL": {
      "required": true
    },
```

Reply:
good catch, we were missing a couple of these, and so they were not inheriting correctly.